LEGO1/lego/legoomni/include/legoracecar.h

```diff
@@ -4,6 +4,39 @@
 #include "legocarraceactor.h"
 #include "legoracemap.h"
 
+struct EdgeReference {
+	char* m_name;
+	LegoPathBoundary* m_data;
+};
+
+// GLOBAL: LEGO1 0x100f0a20
+static EdgeReference g_edgeReferences[] = {
```

Comment:
EDG03_772: This address 0x100f0a20 specifically points to 0x100f0a10, which contains EDG03_772 which I believe is correct.

EDG03_773 has another pointer at 0x100f0a28

Reply:
I did notice those other pointers. Could you make a suggestion how these should be annotated?